index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Portfolio Website</title>
+</head>
+<body>
+  <header id="container">
+    <div class="head">
+        <h1 class="logo">Suleiman</h1>
+    </div>
+    <nav>
+        <img src="images/Hamburguer.png" alt="an image here" class="image">
+    </nav>
+</header>
+
+<section>
+    <h2>I'm Suleiman Welcome To My Portfolio Website</h2>
```

Comment:
- [ ] great job.kindly use an `h1` tag here . this helps for google (and other search engines) knows about a specific topics.